src/natcap/invest/ndr/ndr_core.pyx

```diff
@@ -415,9 +415,9 @@ def ndr_eff_calculation(
 
     # create direction raster in bytes
     def _mfd_to_flow_dir_op(mfd_array):
-        result = numpy.zeros(mfd_array.shape, dtype=numpy.int8)
+        result = numpy.zeros(mfd_array.shape, dtype=numpy.uint8)
```

Comment:
I believe this always should've been `uint8`. I think `result` is supposed to be an 8-digit binary number indicating which of the 8 directions around a pixel have flow. (i.e. `10000001` means there is flow to neighbors 0 and 7). The max value `11111111` = 255, so uint8 is needed. 

I'm not sure exactly why it worked before, but it probably broke because of changes in GDAL's handling of signed 8-bit integers in 3.7.0.